src/Compute/Compute/VirtualMachine/Operation/NewAzureVMCommand.cs

```diff
@@ -54,6 +54,8 @@
 
 namespace Microsoft.Azure.Commands.Compute
 {
+    [GenericBreakingChange("Starting on 10/12/2022 the \"New-AzVM\" cmdlet will deploy with the Trusted Launch configuration by default. To know more about Trusted Launch, please visit https://docs.microsoft.com/en-us/azure/virtual-machines/trusted-launch")]
+    [GenericBreakingChange(" ")]
```

Comment:
```suggestion
    [GenericBreakingChange(" ")]
```
Please remove this extra line

Reply:
Actually, that line was added with the purpose of separating two different Generic Breaking Change warning and making the warnings looks more user readable.

Reply:
@pselugar It would break our tooling that analyze the codebase and generate documents about upcoming breaking changes. Please do remove it.
Please submit a feature request if the output format doesn't look right. Thanks.

Reply:
Ohh okay I understood. Cool I removed the extra spacing